lib/adiwg/mdtranslator/writers/dcat_us/sections/dcat_us_temporal.rb

```diff
@@ -8,7 +8,7 @@ def self.build(intObj)
                   resourceInfo = intObj.dig(:metadata, :resourceInfo)
                   extent = resourceInfo&.dig(:extents, 0)
```

Comment:
FYI it looks like DCAT conversion only cares about the first element in the extents array.

Reply:
right, this is why I modified this writer earlier.  But for now, without changing the writer, we just modified the test xml file which only have one extent. @btylerburton